pkg/api/types_swagger_doc.go

```diff
@@ -0,0 +1,956 @@
+		"spec":   "Spec defines the behavior of a service.",
+		"status": "Status represents the current status of a service.",
+	}
+}
+
+func (ServiceAccount) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                 "ServiceAccount binds together: * a name, understood by users, and perhaps by peripheral systems, for an identity * a principal that can be authenticated and authorized * a set of secrets",
+		"secrets":          "Secrets is the list of secrets allowed to be used by pods running using this ServiceAccount",
+		"imagePullSecrets": "ImagePullSecrets is a list of references to secrets in the same namespace to use for pulling any images in pods that reference this ServiceAccount.  ImagePullSecrets are distinct from Secrets because Secrets can be mounted in the pod, but ImagePullSecrets are only accessed by the kubelet.",
+	}
+}
+
+func (ServiceAccountList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "ServiceAccountList is a list of ServiceAccount objects",
+	}
+}
+
+func (ServiceList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "ServiceList holds a list of services.",
+	}
+}
+
+func (ServicePort) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"name":       "Optional if only one ServicePort is defined on this service: The name of this port within the service.  This must be a DNS_LABEL. All ports within a ServiceSpec must have unique names.  This maps to the 'Name' field in EndpointPort objects.",
+		"protocol":   "The IP protocol for this port.  Supports \"TCP\" and \"UDP\".",
+		"port":       "The port that will be exposed on the service.",
+		"targetPort": "Optional: The target port on pods selected by this service.  If this is a string, it will be looked up as a named port in the target Pod's container ports.  If this is not specified, the default value is the sames as the Port field (an identity map).",
+		"nodePort":   "The port on each node on which this service is exposed. Default is to auto-allocate a port if the ServiceType of this Service requires one.",
+	}
+}
+
+func (ServiceSpec) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                    "ServiceSpec describes the attributes that a user creates on a service",
+		"ports":               "Required: The list of ports that are exposed by this service.",
+		"selector":            "This service will route traffic to pods having labels matching this selector. If empty or not present, the service is assumed to have endpoints set by an external process and Kubernetes will not modify those endpoints.",
+		"clusterIP":           "ClusterIP is usually assigned by the master.  If specified by the user we will try to respect it or else fail the request.  This field can not be changed by updates. Valid values are None, empty string (\"\"), or a valid IP address None can be specified for headless services when proxying is not required",
```

Comment:
"Valid values are None, empty string (\"\"), or a valid IP address", missing the "." at the end.

This occurs because some original comments use "\n" instead of "." at the end of sentence, so I guess there are more occurrences.